docs/source/tracking.rst

```diff
@@ -244,6 +251,13 @@ Running an MLFlow server in ``--artifacts-only`` mode:
   * Listing of artifact responses will pass from the file store through the Tracking Server to the client
   * Loading of artifacts will utilize the access credentials of the MLflow Tracking Server to acquire the files which are then passed on to the client
 
+.. note::
+  - If migrating from Scenario 5 to Scenario 6 due to request volumes, it is important to perform two validations:
+
+    - Ensure that the new tracking server that is operating in ``--artifacts-only`` mode has access permissions to the
```

Comment:
This looks good! On the subject of `--artifacts-only`, I now feel that https://mlflow.org/docs/latest/tracking.html tightly couples the description of proxied logging and `--artifacts-only`, despite this being a less common use case. Could we update the example in https://mlflow.org/docs/latest/tracking.html#mlflow-tracking-servers to just use `--serve-artifacts` without `--artifacts-only` and also update https://mlflow.org/docs/latest/tracking.html#using-the-tracking-server-exclusively-for-proxied-artifact-access so that the section talks about artifact proxying with `--serve-artifacts` (without `--artifacts-only`) and includes a *subsection* discussing `--artifacts-only`? I realize that broadens the scope of this PR, but it should make the content more accessible for users.

Reply:
Great ideas! I de-emphasized the --artifacts-only topics, separated it out into a sub-discussion (smaller header) and focused on the elements that users have expressed confusion on. Examples updated! TY for the great feedback on this - it reads MUCH better now.